src/Page/Guide/View.elm

```diff
@@ -140,7 +139,7 @@ viewRelatedGuideTeasers language guideTitleList allGuidesSlugTitleList =
     in
     if List.length relatedGuideItems > 0 then
         let
-            t : I18n.Keys.Key -> String
+            t : Key -> String
```

Comment:
elm-review doesn't like the import path style that we use elsewhere?

Reply:
No I was bringing this into line with our style for Key elsewhere. We aren't totally consistent on it - but this is the only place it was qualified with `I18n` because there was an accidental import of the `Browser.Navigation.Key` in this file which forced it to be qualified.